.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repos:
+  - repo: https://github.com/pre-commit/mirrors-yapf
+    # this rev is a release tag in the repo above and corresponds with a yapf
+    # version. make sure this matches the version of yapf in tox.ini.
+    rev: v0.29.0
```

Comment:
Please add a similar comment in tox.ini.

Reply:
Please address

Reply:
done!